README.md

```diff
@@ -108,6 +112,119 @@ async onRequestInit({ requestInit }) {
 }
 ```
 
+If you're accepting webhooks from Clerk, you will need to validate the incoming webhook signature. This can be done like this:
+
+```cs
+using Newtonsoft.Json;
+using System.IO;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using Svix;
+using Svix.Models;
+using System.Net;
+
+[ApiController]
+[Route("api/[controller]")]
+public class ClerkController : ControllerBase
+{
+    private readonly IConfiguration _configuration;
+
+    public ClerkController(IConfiguration configuration)
+    {
+        _configuration = configuration;
+    }
+
+    [HttpPost("webhook")]
+    public async Task<IActionResult> ClerkWebhook()
```

Comment:
Inject `IConfiguration` as a function parameter and then delete everything above, reduces the amount of noise being shown.